### ISSUE TITLE
Drag Channel behavior not consistent with permissions

### ISSUE THREAD

**Issue by Grandpa-G** [mumble-voip/mumble#2241]
In Windows/Linux client the following behavior seems to be inappropriate.

If a regular user right clicks on a channel, then drags it to another place, a popup message will ask if you really want to drag the channel. If you say yes, nothing happens, as it should since no permissions are granted to that user to move a channel. If the user doesn't have the permission, then why ask the question?

The action would appear to pop up the question, then check the permission, which seems backwards. This behavior confuses users.

**Comment by bendem** [comment]
My guess is that there is no permission check before moving the channel, just moving it and if it results in a permission error, you didn't have the permission.

**Comment by Grandpa-G** [comment]
Why is this a feature-request. It is a bug. This should not be the correct behavior.

**Comment by hacst** [comment]
@Grandpa-G Just a categorization for us. It's not a bug that it is behaving this way, the server correctly denies the operation and notifies the client of it. Hence adding a more convinient earlier check is a enhancement/feature request for us.

**Comment by Grandpa-G** [comment]
I disagree completely. It is a bug when you indicate you are going to do one thing and then it actually doesn't do it. Your software, do as you wish.

**Comment by hacst** [comment]
Let's agree to disagree then.

(Just to make this clear: We do appreciate reports for such usability issues)
